lib/data_update_scripts/20210423155327_migrate_relevant_fields_from_users_to_users_settings.rb

```diff
@@ -0,0 +1,84 @@
+module DataUpdateScripts
+  class MigrateRelevantFieldsFromUsersToUsersSettings
+    def run
+      ActiveRecord::Base.connection.execute(
+        <<~SQL.squish,
+          BEGIN TRANSACTION;
```

Comment:
If you notice in the terminal when you run the spec you'll see warnings like `WARNING:  there is already a transaction in progress`, this is because statements in tests are wrapped in a transaction that's rolled back at the end, thus `BEGIN transaction` is superflous.

That said, this is meant to run on development and production and, if you load the development console and run this script manually with:

```ruby
require_relative 'lib/data_update_scripts/20210423155327_migrate_relevant_fields_from_users_to_users_settings.rb
DataUpdateScripts::MigrateRelevantFieldsFromUsersToUsersSettings.new.run
```

you'll see how it's actually not wrapped in a transaction.

My suggestion is to use `ActiveRecord::Base.transaction`, this will work both in development:

```ruby
[2] pry(main)> DataUpdateScripts::MigrateRelevantFieldsFromUsersToUsersSettings.new.run
  TRANSACTION (0.4ms)  BEGIN
   (153.2ms)  WITH settings_data AS ...
  TRANSACTION (8.5ms)  COMMIT
```

and in the tests.

Something like:

```ruby
      ActiveRecord::Base.transaction do
        ActiveRecord::Base.connection.execute(...)
      end
```

Reply:
@rhymes Do I also remove the `COMMIT;` at the end of the SQL statement?

Reply:
@msarit Yes, thank you!